lists/th.csv

```diff
@@ -456,7 +456,7 @@ https://prd.go.th/,NEWS,News Media,2022-10-09,thainetizen,Government Public Rela
 https://live.parliament.go.th/,NEWS,News Media,2022-10-09,thainetizen,Parliament TV. A free-to-air terrestrial television. State-owned. [th]
 https://mono29.com/,NEWS,News Media,2022-10-09,thainetizen,Mono 29. A free-to-air terrestrial television. Privately owned. [th]
 https://www.one31.net/,NEWS,News Media,2022-10-09,thainetizen,One Channel. A free-to-air terrestrial television. Privately owned. [th]
-https://www.benarnews.org/,NEWS,News Media,2022-10-09,thainetizen,"Benar News - news on security, politics, and human rights - serving Philippines, Indonesia, Malaysia, Thailand and Bangladesh. [th] [en] [id] [ms] [bn]"
+https://www.benarnews.org/thai,NEWS,News Media,2022-10-09,thainetizen,"Benar News - news on security, politics, and human rights - serving Philippines, Indonesia, Malaysia, Thailand and Bangladesh. [th] [en] [id] [ms] [bn]"
```

Comment:
Ideally this would be dropped from the country list, since it's now in the global list